### DISCUSSION TITLE
old docs need some love

### DISCUSSION THREAD

**Discussion by Inukares** [mitchelvanbever/remix-auth-supabase#34] [General]
Docs state 
```ts
// app/auth.server.ts
import { createCookieSessionStorage } from 'remix';
import { Authenticator, AuthorizationError } from 'remix-auth';
import { SupabaseStrategy } from '@afaik/remix-auth-supabase-strategy';
```

but 'remix' nowadays should be '@remix-run/node', '@afaik/remix-auth-supabase-strategy' should be 'remix-auth-supabase', etc.

PS: Love the package!

**Comment by mitchelvanbever** [comment]
Hi there, 

Thanks for the love 💜 

Seems indeed there are a few places where the docs could use some love and attention 😉 Thanks for making me aware 👍 

PR's are welcome otherwise I'll have a go at it this weekend 👌

**Comment by mitchelvanbever** [comment]
It's been a while but the docs are updated.

Now onto Supabase V2 as pointed out in #43 😁
